evalml/automl/automl_search.py

```diff
@@ -124,9 +124,6 @@ class AutoMLSearch:
 
     _MAX_NAME_LEN = 40
 
-    # Necessary for "Plotting" documentation, since Sphinx does not work well with instance attributes.
-    plot = PipelineSearchPlots
```

Comment:
`plot` shows up as an attribute in the docs without this

https://feature-labs-inc-evalml--2458.com.readthedocs.build/en/2458/autoapi/evalml/automl/index.html#evalml.automl.AutoMLSearch.plot